hermes-common/src/main/java/pl/allegro/tech/hermes/common/di/factories/MetricRegistryFactory.java

```diff
@@ -34,15 +32,14 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
-public class MetricRegistryFactory implements Factory<MetricRegistry> {
+public class MetricRegistryFactory {
```

Comment:
I would get rid of these factories. In my opinion, they duplicate that we do in `*Configuration` classes.